src/stdune/user_message.ml

```diff
@@ -0,0 +1,116 @@
+module Style = struct
+  type t =
```

Comment:
Oh, I guess there's some relationship between this and the code in src/colors.ml, but it's not clear to me what the relationship is. Is the plan to get rid of src/colors.ml then?

Reply:
Yep. I updated src/colors.ml to remove a bit of the duplication